dev_guide/create_from_url.adoc

```diff
@@ -0,0 +1,81 @@
+[[dev-guide-create-from-url]]
+= Create From URL
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+[[overview]]
+== Overview
+
+Create from URL allows the construction of a URL that drops the user into the end
+(or near end) of the creation flow using an image stream and image tag or
+template.
+
+Create from URL only works with image streams or templates from
+namespaces that have been explicitly whitelisted.  The whitelist contains
+the 'openshift' namespace by default.  To add namespaces to the whitelist, see
+xref:../install_config/web_console_customization.adoc#configuring-the-create-
+from-url-namespace-whitelist[Configuring the Create From URL Namespace Whitelist].
+
+[[using-an-image-stream-and-image-tag]]
+== Using an Image Stream and Image Tag
+
+=== Query String Parameters
```

Comment:
Each heading needs a unique ID before it so that the docs build properly, so for Query String Parameters:
[[image-stream-and-tag-query-string-parameters]]